vsintegration/src/FSharp.ProjectSystem.Base/Project/ProjectNode.CopyPaste.cs

```diff
@@ -39,11 +39,6 @@ public override int DragEnter(IOleDataObject pDataObject, uint grfKeyState, uint
         {            
             pdwEffect = (uint)DropEffect.None;
 
-            if (this.SourceDraggedOrCutOrCopied)
```

Comment:
@KevinRansom you're shown as the last person to touch this part of the code...any concerns?